### ISSUE TITLE
[Bug] MangaDex images are low quality compared to Tachiyomi

### ISSUE THREAD

**Issue by rohan-gt** [zyrouge/yukino-app#10]
Not sure what's the reason but MangaDex images in Yukino are compressed and low quality. Can it be made to fetch the original quality images?

**Comment by zyrouge** [comment]
Are you sure? The resolution seems fine for me. 
![image](https://user-images.githubusercontent.com/50770114/122045089-e906ff00-cdfa-11eb-87f2-71078054e1ba.png)

**Comment by rohan-gt** [comment]
Yes. Try downloading the cover image of any manga from the two sources and comparing the file size. There is some lossy compression

**Comment by zyrouge** [comment]
Oh yeah, the current "MangaDex" is https://mangadex.tv and not the https://mangadex.org or https://api.mangadex.org, but the .tv doesn't seem bad (might be impersonation official MangaDex but seems legit). I'll be adding MangaDex API or the offical site scraper!

**Comment by Ryan451** [comment]
https://www.reddit.com/r/mangadex/comments/ljore6/what_is_the_difference_between_mangadexorg_and/
